doc-requirements.txt

```diff
@@ -1,29 +1,31 @@
 #
-# This file is autogenerated by pip-compile (and then all packages are manually
-# unpinned to use the versions in katsdpdockerbase base-requirements.txt).
-#
+# This file is autogenerated by pip-compile
 # To update, run:
 #
-#    pip-compile --output-file doc-requirements.txt doc-requirements.in
+#    pip-compile doc-requirements.in
 #
-alabaster                   # via sphinx
-babel                       # via sphinx
-certifi                     # via requests
-chardet                     # via requests
-docutils                    # via sphinx
-idna                        # via requests
-imagesize                   # via sphinx
-jinja2                      # via sphinx
-markupsafe                  # via jinja2
-packaging                   # via sphinx
-pygments                    # via sphinx
-pyparsing                   # via packaging
-pytz                        # via babel
-requests                    # via sphinx
-six                         # via packaging, sphinx
-snowballstemmer             # via sphinx
-sphinx-rtd-theme
-sphinx
-sphinxcontrib-websupport    # via sphinx
-typing; python_version<'3'  # via sphinx
-urllib3                     # via requests
+alabaster==0.7.12         # via sphinx
```

Comment:
I thought the reason we unpinned this is because they match the docker-base versions?

Reply:
Yes, but readthedocs doesn't know anything about docker-base, so we have to explicitly pin if we want readthedocs and Jenkins to build docs from the same versions.